hal/lib/athena/Accelerometer.cpp

```diff
@@ -25,7 +25,7 @@ static const uint8_t kControlTxRx = 1;
 static const uint8_t kControlStart = 2;
 static const uint8_t kControlStop = 4;
 
-static tAccel* accel = 0;
+static unique_ptr<tAccel> accel;
```

Comment:
Yeah I need to fix this and something else. Apparently I forgot to even build this one....